packages/components/src/Combobox/ComboboxProvider.tsx

```diff
@@ -11,16 +12,20 @@ export const ComboboxContext = React.createContext(
 
 export interface ComboboxProviderProps {
   readonly children: React.ReactNode;
+  readonly multiselect?: boolean;
```

Comment:
we've defined the prop for multiselect on the provider, but nothing is passing that value yet. let's add that small change so any instances doing `<Combobox multiselect />` truly have `multiselect` available to them inside `Combobox` via the Provider